resources/assets/icons/question-mark.svg

```diff
@@ -1 +1 @@
-<svg version="1.1" id="Hint_2_1_" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 14.5 10" xml:space="preserve"><style>.question-mark-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round;stroke-linejoin:round}</style><g id="Hint_2"><path id="Shape_895_1_" class="question-mark-st0" d="M7.2 8.6c.1 0 .2.1.2.2s-.1.2-.2.2-.1-.1-.1-.2c-.1-.1 0-.2.1-.2"/><path id="Shape_896_1_" class="question-mark-st0" d="M5.4 2.8c0-1 .8-1.8 1.8-1.8S9 1.8 9 2.8c0 .8-.5 1.4-1.2 1.7-.4.1-.6.5-.6.8v.4"/></g></svg>
\ No newline at end of file
+<svg version="1.1" id="question-mark-small" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 10 10" xml:space="preserve"><path d="M5 9h0M3.2 2.8C3.2 1.9 3.9 1 5 1s1.8.8 1.8 1.8c0 .8-.4 1.5-1.3 1.7-.3.2-.5.5-.5.8v.9" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
I take it this is for nodem? if so, how large are the other icons? since this one has a viewbox of 10x10 and will be out of line with the others when those have 20x20

Reply:
This is for explorer (updated the PR description). I think this question mark icon might intentionally have a smaller viewbox as it's a smaller icon. the name originally when Oleg sent it was `question-mark-small` (looks like it'sß part of the `id` too. Is it worth me renaming or leaving as it was already roughly that size viewbox before? 🤔

Reply:
can leave it as it was similar, was more curious for nodem if the icon was used at this size too or not